stats/aggregation.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package stats defines the stats collection API and its native Go
+// implementation.
```

Comment:
This is a Go package, you don't have to say it is a Go implementation. It is redundant.

Reply:
Done

Reply:
I would but it doesn't exist yet.